### PR TITLE
Moby CVEs trivyignore

### PR DESCRIPTION
**Pull request by bewebi**
# Description

- Add two CVEs for vulnerabilities in `moby/moby`

# Context

See justification [here](https://github.com/solo-io/solo-projects/issues/7348#issuecomment-2518727997)

## Notes for reviewers

This is marked as "work in progress" until consensus is reached on this path forward

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
BOT NOTES: 
resolves https://github.com/solo-io/solo-projects/issues/7357
resolves https://github.com/solo-io/solo-projects/issues/7358
resolves https://github.com/solo-io/solo-projects/issues/7359

### CODE REVIEW

**Comment by danehans** [comment]
The `Kubernetes Tests / End-to-End (cluster-six) (pull_request)` job is failing due to https://github.com/solo-io/gloo/issues/10397. I'll rekick the job. If the failure persists, please rebase.

File: .trivyignore [hunk 1]
```diff
@@ -67,4 +67,14 @@ CVE-2024-27304
...
+
+# https://github.com/advisories/GHSA-2mj3-vfvx-fc43
+# https://github.com/advisories/GHSA-gh5c-3h97-2f3q
+# These are not expected to impact us and are difficult to resolve due to breaking API changes that impact WASM plugin
+# code
+# More detail can be found here: https://github.com/solo-io/solo-projects/issues/7348#issuecomment-2518727997
+# We can remove these once moby/moby has been upgrade to v26+ on all LTS branches and/or the WASM plugin has been
+# removed from all LTS branches
+CVE-2024-36621
+CVE-2024-36623
```
Comment (danehans) [inline .trivyignore:80]: nit: should this file end with a new line?

File: changelog/v1.19.0-beta1/moby-cves-trivyignore.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    issueLink: https://github.com/solo-io/solo-projects/issues/7353
+  - type: NON_USER_FACING
+    description: This also resolves the issue for 1.16
+    issueLink: https://github.com/solo-io/solo-projects/issues/7357
+  - type: NON_USER_FACING
+    description: This also resolves the issue for 1.15
+    issueLink: https://github.com/solo-io/solo-projects/issues/7358
+  - type: NON_USER_FACING
+    description: This also resolves the issue for 1.14
+    issueLink: https://github.com/solo-io/solo-projects/issues/7359
```
Comment (danehans) [inline changelog/v1.19.0-beta1/moby-cves-trivyignore.yaml:13]: nit: should this file end with a new line?

**Review by danehans — APPROVED** [review]

**Comment by bewebi** [comment]
I'm investigating the alternative solution proposed by @sam-heilbron [here](https://github.com/solo-io/solo-projects/issues/7348#issuecomment-2519219511) which may eliminate the need for this PR

**Comment by sam-heilbron** [comment]
@bewebi are we safe to close this?

**Comment by bewebi** [comment]
We ran into complications backporting the EE CVE fix to 1.16 and, while that PR (solo-io/solo-projects#7396) is now passing CI, it required some dep bumps which may cause issues in 1.15 and 1.14 (eg bumping Go to 1.22)

Therefore I'd advocate to leave this open until those backports are done and we know there is no need for this
On the other hand, if we close this PR and later discover a need, we can always reopen it

File: .trivyignore [hunk 3]
```diff
@@ -67,4 +67,14 @@ CVE-2024-27304
...
 # and captured our findings here: https://github.com/solo-io/solo-projects/issues/7157#issuecomment-2463252858
-CVE-2022-30635
\ No newline at end of file
+CVE-2022-30635
+
+# https://github.com/advisories/GHSA-2mj3-vfvx-fc43
+# https://github.com/advisories/GHSA-gh5c-3h97-2f3q
+# These are not expected to impact us and are difficult to resolve due to breaking API changes that impact our
+# code
+# While this has been resolved on v1.16+, backporting it to lower versions is complicated and we opted to skip it
```
Comment (jenshu) [inline .trivyignore:76]: should it be resolved in 1.17+ since we are reverting the fix in EE 1.16?

**Review by ashishb-solo — APPROVED** [review]

File: .trivyignore (hunk 3 above)
Comment (bewebi) [reply .trivyignore:76]: yup

**Review by ashishb-solo — APPROVED** [review]

**Pull request merged**